backbone/src/main/java/org/researchstack/backbone/ui/task/TaskViewModel.kt

```diff
@@ -238,4 +251,21 @@ internal class TaskViewModel(context: Application, intent: Intent) : AndroidView
     }
 
 
+    fun showCancelEditAlert() {
+        showEditDialog.postValue(true)
+    }
+
+    fun cancelEditDismiss() {
+        stack.push(currentStep)
+    }
+
+
+    fun removeUpdatedLayout() {
+        updateCancelEditInLayout.postValue(true)
+        goToReviewStep()
+        stack.clear()
+        editing = false
+    }
+
+
```

Comment:
extra space here too (basically check that there aren't new double spaces between things, thanks!)